app/services/analytics_events.rb

```diff
@@ -893,14 +893,14 @@ def idv_doc_auth_document_capture_visited(**extra)
   end
 
   # @param [String] step_name which step the user was on
-  # @param [Integer] remaining_attempts how many attempts the user has left before
+  # @param [Integer] remaining_submit_attempts how many attempts the user has left before
```

Comment:
ditto about previous name
```suggestion
  # @param [Integer] remaining_submit_attempts how many attempts the user has left before (previously called "remaining_attempts")
```